Lanchonete/LanchoneteTests/Managers/ShoppingCartManagerTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ShoppingCartManagerTests.swift
+//  LanchoneteTests
+//
+//  Created by vera.dias on 1/22/21.
+//
+
+import XCTest
+@testable import Lanchonete
+
+class ShoppingCartManagerTests: XCTestCase {
+    
+    func test_GIVEN_an_empty_shopping_cart_WHEN_I_add_a_sandwich_THEN_the_total_should_be_the_price_of_the_added_sandwich() {
+        let manager = ShoppingCartManager()
+        //let sandwich = Sandwich(type: .xBacon)
```

Comment:
I believe if we are not going to use this code right now it's better to just remove the line instead of commenting this out. It will help to keep the code base clean.

Reply:
Thank you Paulo, I'll remove it now.